pkg/kv/kv.go

```diff
@@ -689,6 +689,8 @@ type Storage interface {
 	UUID() string
 	// CurrentVersion returns current max committed version with the given txnScope (local or global).
 	CurrentVersion(txnScope string) (Version, error)
+	// CurrentMinTimestamp returns current minimum ts for all keyspace groups
+	CurrentMinTimestamp() (uint64, error)
```

Comment:
Can the name be clearer, such as something like `GetMinTimestampAmongAllKeyspaces`? Also consider update the naming in client-go.
I think the naming is not hard enough that you have nothing to do but explain it in comments.